pulser-core/pulser/backend/observable.py

```diff
@@ -99,6 +99,16 @@ def __init__(
     def _base_tag(self) -> str:
         pass
 
+    def _to_abstract_repr(self) -> dict[str, Any]:
+        return {
+            self.tag: {
+                self._base_tag: {
+                    "evaluation_times": self.evaluation_times,
+                    "tag_suffix": self._tag_suffix,
+                }
+            }
+        }
```

Comment:
I understand why you did this but this kind of nesting is not very nice on the schema. In fact, I'd argue that an observable should be serialized as a flat object, ie
```suggestion
    def _to_abstract_repr(self) -> dict[str, Any]:
        return {
                "observable": self._base_tag,
                "evaluation_times": self.evaluation_times,
                "tag_suffix": self._tag_suffix,
        }
```

Reply:
yes, I agree that it's preferable to just serialize the object as a flat dict, so you can deserialize it as
`Observable(**serialized_dict)`